### DISCUSSION TITLE
Improve country level maps

### DISCUSSION THREAD

**Discussion by arky** [neocarto/bertin#36] [Q&A] [answered]
The country level boundary using the bertin geojson data tend to look spiky. Is there any way to improve the shapes of the country boundaries?

![image](https://user-images.githubusercontent.com/808895/159493870-6b9e8dab-ef04-4495-9f71-46642bfacfab.png)

**Comment by neocarto** [comment, marked as answer]
In my opinion, this issue of cartographic generalization is outside the package.

**Comment by arky** [comment]
Agreed. However it would be good to provide some base maps that are usable out of the box. 

I like the idea of quickly creating a map with some standard defaults. Like what Folium does. 

```python 
import folium


m = folium.Map(location=[45.5236, -122.6750])

m 
```

https://python-visualization.github.io/folium/quickstart.html

**Reply by neocarto** [reply]
There are already a lot of free basemaps available. 
https://ec.europa.eu/eurostat/fr/web/gisco/geodata/reference-data/administrative-units-statistical-units/countries
